packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -89,7 +104,17 @@ export type UIEventConfig<T = EventHandlerMap> = {
     // Whether to cache element's react information on capture, defaults to false.
     cacheElementReactInfo?: boolean;
   }>
-}[keyof T];
+}[keyof T] | {
+  eventName: 'hover',
+  // A callable filter for this event, returning true if the event should be emitted, or false if it should be discarded up front
+  eventFilter?: (domEvent: MouseEvent) => boolean;
+  // Whether to limit to elements that are "interactable", i.e. those that have event handlers registered to the element.  Defaults to true.
+  interactableElementsOnly?: boolean;
```

Comment:
at the moment some of this configuration is not applied to `hover` but is required or causes type issues in many other places,  even though the code complaining about the type will never be reached the way it is initialized currently